application/src/test/java/org/opentripplanner/generate/doc/GtfsRtUpdaterConfigDocTest.java

```diff
@@ -23,21 +23,17 @@
 import org.opentripplanner.standalone.config.framework.json.NodeAdapter;
 
 @GeneratesDocumentation
-public class UpdaterConfigDocTest {
+public class GtfsRtUpdaterConfigDocTest {
 
-  private static final File TEMPLATE = new File(TEMPLATE_PATH, "UpdaterConfig.md");
-  private static final File OUT_FILE = new File(USER_DOC_PATH, "UpdaterConfig.md");
+  private static final File TEMPLATE = new File(TEMPLATE_PATH, "GTFS-RT-Config.md");
+  private static final File OUT_FILE = new File(USER_DOC_PATH, "GTFS-RT-Config.md");
 
   private static final String ROUTER_CONFIG_PATH = "standalone/config/" + ROUTER_CONFIG_FILENAME;
-  private static final Set<String> SKIP_UPDATERS = Set.of(
-    "siri-azure-sx-updater",
-    "siri-azure-et-updater",
-    "vehicle-parking",
```

Comment:
Btw how do these changes affect vehicle-parking documentation?

Reply:
They don't affect it at all. This just changes from a skip list to a whitelist.